.github/workflows/build-rpm.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         run: mkdir -p ~/.config/
```

Comment:
Do the lines above for `./code/packaging/rpm/go-ipfs.spec` need to be changed to use kubo?